From 54f6b86f81670866d6116734137643db27cd58ff Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Fri, 2 Feb 2024 14:14:47 +0000
Subject: [PATCH 0881/1085] overlays: Delete deprecated overlay mpu6050

The mpu6050 overlay has been deprecated for a year (when we were still
shipping rpi-5.15.y). Delete it.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/Makefile           |  1 -
 .../arm/boot/dts/overlays/mpu6050-overlay.dts | 29 -------------------
 2 files changed, 30 deletions(-)
 delete mode 100644 arch/arm/boot/dts/overlays/mpu6050-overlay.dts

--- a/arch/arm/boot/dts/overlays/Makefile
+++ b/arch/arm/boot/dts/overlays/Makefile
@@ -173,7 +173,6 @@ dtbo-$(CONFIG_ARCH_BCM2835) += \
 	mipi-dbi-spi.dtbo \
 	mlx90640.dtbo \
 	mmc.dtbo \
-	mpu6050.dtbo \
 	mz61581.dtbo \
 	ov2311.dtbo \
 	ov5647.dtbo \
--- a/arch/arm/boot/dts/overlays/mpu6050-overlay.dts
+++ /dev/null
@@ -1,29 +0,0 @@
-// Definitions for MPU6050
-/dts-v1/;
-/plugin/;
-
-/ {
-        compatible = "brcm,bcm2835";
-
-        fragment@0 {
-                target = <&i2c1>;
-                __overlay__ {
-                        #address-cells = <1>;
-                        #size-cells = <0>;
-                        status = "okay";
-                        clock-frequency = <400000>;
-
-                        mpu6050: mpu6050@68 {
-                                compatible = "invensense,mpu6050";
-                                reg = <0x68>;
-                                interrupt-parent = <&gpio>;
-                                interrupts = <4 1>;
-                        };
-                };
-        };
-
-        __overrides__ {
-                interrupt = <&mpu6050>,"interrupts:0";
-                addr = <&mpu6050>,"reg:0";
-        };
-};
